src/components/Dashboard/TodoList.vue

```diff
@@ -431,6 +456,10 @@ export default {
       return sessionStorage.getItem('BUSINESS_ID')
     },
 
+    payAPiUrl (): string {
```

Comment:
This computed method is now obsolete.

Reply:
it's also named incorrectly :p will remove

Reply:
Fixed.